docs/data/install-xrootd.md

```diff
@@ -290,6 +290,70 @@ fermicloud054.fnal.gov complete inventory as of Tue Apr 12 07:38:29 2011 /data/x
 
 **Note**: In this example, `fermicloud53.fnal.gov` is a redirector node and `fermicloud054.fnal.gov` is a data node.
 
+### (Optional) Enabling Xrootd over HTTP
+
+XRootD can be accessed using the HTTP protocol. To do that:
+
+1.   Modify `/etc/xrootd/xrootd-clustered.cfg` and add the following lines:
+
+        :::file
+           sec.protocol /usr/lib64 gsi \
+              -certdir:/etc/grid-security/certificates \
+              -cert:/etc/grid-security/xrd/xrdcert.pem \
+              -key:/etc/grid-security/xrd/xrdkey.pem \
+              -crl:3 \
+              -authzfun:libXrdLcmaps.so \
+              -authzfunparms:--lcmapscfg,/etc/xrootd/lcmaps.cfg,--loglevel,4 \
+              -gmapopt:10 \
+              -gmapto:0
+           if exec xrootd
```

Comment:
What happens if `if exec xrootd` is left out?  Does it mess up the `cmsd`?  Or is this a line we've been copy/pasting for years and never was really required?

Reply:
Yep it does:

```180802 14:15:02 27686 Update Space Parm1=0 Parm2=0
180802 14:15:02 27686 Meter: Write access and staging prohibited.
------ server@gftp-2.t2.ucsd.edu phase 2 server initialization completed.
180802 14:15:02 27686 XrdOpen: Unable to bind socket to port 1094; address already in use
------ cmsd server@gftp-2.t2.ucsd.edu:1094 initialization failed.````